pallets/automation-time/src/lib.rs

```diff
@@ -588,15 +586,15 @@ pub mod pallet {
 								),
 						};
 						Tasks::<T>::remove(task_id);
-						action_weight + 10_000
+						<T as Config>::WeightInfo::run_tasks_many_found(1)
 					},
 				};
 
 				// need to calculate the look cost minus the action
-				weight_left = weight_left - action_weight - 10_000;
+				weight_left = weight_left - action_weight;
 
 				// need to calculate the max cost of the loop
-				if weight_left < 20_000 {
+				if weight_left < <T as Config>::WeightInfo::run_tasks_many_found(1) {
```

Comment:
just to double check my sanity here, max of loop should just be run_tasks_many_found, right?